auditbeat/docs/index.asciidoc

```diff
@@ -17,6 +17,8 @@ include::../../libbeat/docs/contributing-to-beats.asciidoc[]
 
 include::./getting-started.asciidoc[]
 
+include::./breaking.asciidoc[]
```

Comment:
Didn't notice this the first time I reviewed, but the include for breaking changes needs to come after the repositories topic (line 22), or you end up with a TOC that looks like this:

![image](https://user-images.githubusercontent.com/14206422/35263221-c47507d6-ffcc-11e7-84d0-1561db352f48.png)